### PR TITLE
[battery_manager] maximum value in RB is too high. #9079 - fix 

### PR DESCRIPTION
**Pull request by kongtiaowang**
[battery_manager] maximum value in RB is too high. #9079

### CODE REVIEW

File: raisinbread/RB_files/RB_test_battery.sql [hunk 1]
```diff
@@ -1,38 +1,39 @@
 SET FOREIGN_KEY_CHECKS=0;
 TRUNCATE TABLE `test_battery`;
 LOCK TABLES `test_battery` WRITE;
-INSERT INTO `test_battery` (`ID`, `Test_name`, `AgeMinDays`, `AgeMaxDays`, `Active`, `Stage`, `CohortID`, `Visit_label`, `CenterID`, `firstVisit`, `instr_order`) VALUES (131,'aosi',1,2147483647,'Y','Visit',NULL,'V1',NULL,NULL,NULL);
+-- AgeMaxDays maximum value is 99999
+INSERT INTO `test_battery` (`ID`, `Test_name`, `AgeMinDays`, `AgeMaxDays`, `Active`, `Stage`, `CohortID`, `Visit_label`, `CenterID`, `firstVisit`, `instr_order`) VALUES (131,'aosi',1,36500,'Y','Visit',NULL,'V1',NULL,NULL,NULL);
 INSERT INTO `test_battery` (`ID`, `Test_name`, `AgeMinDays`, `AgeMaxDays`, `Active`, `Stage`, `CohortID`, `Visit_label`, `CenterID`, `firstVisit`, `instr_order`) VALUES (132,'aosi',1,4299,'Y','Visit',NULL,'V2',NULL,NULL,NULL);
```
Comment (regisoc) [inline raisinbread/RB_files/RB_test_battery.sql:6]: I do not know AOSI instrument, is there any particular reason for this one to stay 4299?

Comment (kongtiaowang) [reply raisinbread/RB_files/RB_test_battery.sql:6]: ok, I will fix it to 36500 as well.

Comment (driusan) [reply raisinbread/RB_files/RB_test_battery.sql]: It's already below the limit, is there any reason to change it? (it's the "Autism Observation Scale for Infants" and 4299 works out to just under 12yo so I don't think there's any reason to raise it.)

Comment (regisoc) [reply raisinbread/RB_files/RB_test_battery.sql]: Ok I did not know what the acronym stood for. Thanks. I think 100y is enough for the rest.

**Comment by regisoc** [comment]
@kongtiaowang sorry for that, can you change it back to `4299`? It was making sense for a child focused instrument.

**Review by CamilleBeau — CHANGES_REQUESTED** [review]
I think the ageMaxDays default should be a little bit higher than 100 years old (excluding aosi or similar instruments). While it's rare, it is possible to have people older than 100 in the database. I would go with 110 or 120 to be safe (43800)

**Comment by kongtiaowang** [comment]
@regisoc @CamilleBeau it is done.

**Pull request merged**